ui/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "ems-ui",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
```

Comment:
if aot is not enabled by default in angular 8.x, you have to enable it by using ng build --prod. This might throw some errors in your application when enabled but required.